data/assets/scene/digitaluniverse/constellations.asset

```diff
@@ -35,47 +35,47 @@ local function zodiacsString(zodiacsList)
   return zodiacsString
 end
 
-local ConstellationsExtragalactic = {
-  Identifier = "ConstellationsExtragalactic",
-  Renderable = {
-    Type = "RenderableConstellationLines",
-    Enabled = false,
-    Labels = {
-      File = speck .. "constellationsEXGAL.label",
-      Opacity = 0.4,
-      Color = { 0.8, 0.8, 0.8 },
-      Size = 20.0,
-      MinMaxSize = { 20, 30 },
-      Unit = "Mpc"
-    },
-    Opacity = 0.4,
-    File = speck .. "constellationsEXGAL.speck",
-    NamesFile = speck .. "constellations.dat",
-    Colors = { { 0.6, 0.4, 0.4 }, { 0.8, 0.0, 0.0 }, { 0.0, 0.3, 0.8 } },
-    Unit = "Mpc",
-    -- Selection = zodiacs
-  },
-  GUI = {
-    Name = "Constellations (Extragalactic)",
-    Path = "/Milky Way/Constellations"
-  }
-}
+-- local ConstellationsExtragalactic = {
```

Comment:
I'b be curious about how anyone uses this. Curatorially (if that's a word) I'd really like to remove it. It made sense when we couldn't traverse from the Milky Way to the Extragalactic without restarting, now I struggle to see the use for it because if you want to see the galaxies in concert with the constellations, one need only go home and force the galaxies to be on.

Reply:
I think we should remove it. I'm always confused what the extra galactic constellations are, and they seem like a "dumber" version than the regular constellations.

Reply:
If we decide to keep them, we should set the Opacity of the regular constellations and the extragalactic ones to the same value.